source/exec.tex

```diff
@@ -6006,9 +6030,9 @@
 %%FIXME: Was this supposed to be more than a single bullet?
 \begin{itemize}
 \item
-Uses \tcode{alloc} to allocate and construct an object \tcode{o} of
-type that is a specialization of \tcode{\exposid{spawn-state}} from
-\tcode{alloc}, \tcode{write_env(token.wrap(sndr), senv)}, and \tcode{token}
+Uses \tcode{alloc} to allocate and construct an object \tcode{o} of type
+\tcode{decltype(\exposid{spawn-state}(alloc, write_env(token.wrap(sndr), senv), token))}
```

Comment:
This adds a closing parenthesis which wasn't in the paper, but that's a fix.

Reply:
It's fine like this, I was just noting it as part of the review to confirm that the deviation from the paper is desirable. No need for a separate commit.